aws/resource_aws_ecs_service.go

```diff
@@ -354,7 +348,7 @@ func resourceAwsEcsServiceCreate(d *schema.ResourceData, meta interface{}) error
 	input.SchedulingStrategy = aws.String(schedulingStrategy)
 	if schedulingStrategy == ecs.SchedulingStrategyDaemon {
 		// unset these if DAEMON
-		input.DeploymentConfiguration = nil
+		input.DeploymentConfiguration.MaximumPercent = aws.Int64(100)
```

Comment:
Is there a reason to keep this logic here at all? Since the comment eludes to "unsetting" values due to `DAEMON` we should either remove this line or add a comment why its being hardcoded to `100`

Reply:
I updated this to be `input.DeploymentConfiguration.MaximumPercent = nil` as well